src/Tempest/Http/src/Mappers/PsrRequestToRequestMapper.php

```diff
@@ -53,7 +53,7 @@ public function map(mixed $from, mixed $to): array|object
             'path' => $from->getUri()->getPath(),
             'query' => $query,
             'files' => $uploads,
-            ...$data,
+            ...$data + $uploads,
```

Comment:
Could you write a test?

Reply:
Also, I'm personally more a fan on `[...$data, ...$uploads]`, since `[] + []` works a bit weird in PHP

Reply:
OK. can I use the `FormRequest` for testing?